tests/test_ensdf_processing.py

```diff
@@ -163,8 +226,8 @@ def file_comp(file_out, file_ref, exceptions):
         type 4: carriage return vs. non standard return type.
             options: line number of return.
     '''
-    f_out = open(file_out, 'r')
-    f_ref = open(file_ref, 'r')
+    f_out = open(file_out, 'r',encoding="latin-1")
+    f_ref = open(file_ref, 'r',encoding="latin-1")
```

Comment:
@scopatz I'm pretty sure the bldhst analysis program writes its output with some other encoding.  When utf8 or ascii are used, a couple of invalid continuation byte errors occur when trying to read in the file for comparison.  The only encoding I could get to work with bldhst was 'latin-1.'